interoperator/pkg/controller/sfservicebinding/sfservicebinding_controller.go

```diff
@@ -322,11 +324,17 @@ func (r *ReconcileSFServiceBinding) updateUnbindStatus(targetClient client.Clien
 	bindingID := binding.GetName()
 	namespace := binding.GetNamespace()
 	computedStatus, err := r.resourceManager.ComputeStatus(r, targetClient, instanceID, bindingID, serviceID, planID, osbv1alpha1.BindAction, namespace)
-	if err != nil {
+	if err != nil && !errors.IsNotFound(err) {
 		log.Error(err, "ComputeStatus failed for unbind", "binding", bindingID)
 		return err
 	}
 
+	if errors.IsNotFound(err) && computedStatus == nil {
```

Comment:
What does this block of code do? Shouldn't we delete the binding crd if operator bind crd is not present?

Reply:
This is in the updateUnbindStatus function. This means delete was already triggered. We just have to not fail the delete here